Automattic/jetpack
# Bump the jetpack blocks version to 6.8.0-beta.1

Bumps the version of jetpack-blocks to 6.8.0-beta.1 release. 


#### Changes proposed in this Pull Request:

*

#### Testing instructions:
run `yarn build` 
Does the markdown block work as expected.
Can you turn it on and off ? Via the module activation.
`yarn docker:wp jetpack module activate markdown`
